### ISSUE TITLE
Docstring of SFrame.print_rows evaluates sys.stdout

### ISSUE THREAD

**Issue by rschindlauer** [dato-code/SFrame#106]
The docstring signature of `SFrame.print_rows` is supposed to look like this:
```
def print_rows(self, num_rows=10, num_columns=40, max_column_width=30,
               max_row_width=80, output_file=sys.stdout):
```
However, looking at the docstring in the python shell it looks like `sys.stdout` somehow got evaluated:
```
print_rows(self, num_rows=10, num_columns=40, max_column_width=30,
           max_row_width=80, output_file=<open file '<stdout>', mode 'w'>) unbound sframe.data_structures.sframe.SFrame method
```

**Comment by MatthewWest** [comment]
This may be unavoidable. Python evaluates default arguments once at definition time, rather than keeping them as pieces of code to be evaluated at call time (see [this](http://stackoverflow.com/questions/1651154/why-are-default-arguments-evaluated-at-definition-time-in-python) Stack Overflow question for more discussion).

One method which appears to be a solution would be monkey-patching the `__repr__` method on `sys.stdout`, but from a cursory examination it looks like this might not be possible (see [this](http://stackoverflow.com/questions/26050557/python-why-does-a-dynamically-added-repr-method-not-override-the-default) Stack Overflow question for details).

**Comment by znation** [comment]
Fixed with 6216ad1
